### DISCUSSION TITLE
Secret network

### DISCUSSION THREAD

**Discussion by viriava** [eco-stake/restake#496] [Q&A] [answered]
Hey guys!

When I try to retake I receive the error: Failed to broadcast: Broadcasting transaction failed with code 13 (codespace: sdk). Log: insufficient fees; got: 780uscrt required: 3898uscrt: insufficient fee

https://restake.app/secretnetwork

I have coins to pay fees https://tppr.me/YkU8

Can you help?

Thank you

**Comment by tombeynon** [comment, marked as answer]
Just seen this! Secret was updated in REStake to use a higher minimum gas price, so this should have been resolved now. Can you confirm?

FYI the manual fix is to increase the gas in the Keplr pop up.
